deploy/autopg.Dockerfile

```
@@ -0,0 +1,96 @@
+# Must match PINNED_PG_VERSION in autopg's src/postgres.js for the
+# AUTOPG_VERSION above (v3.0.7 pins 18.3.0-beta.17). Bump both together.
+ARG PG_RUNTIME_VERSION=18.3.0-beta.17
+
+# ---- fetch: download, verify, and unpack the release tarball ----------------
+# Runs on the build platform (no target-arch execution — pure download/copy).
+FROM --platform=$BUILDPLATFORM debian:bookworm-slim AS fetch
+ARG AUTOPG_VERSION
+ARG PG_RUNTIME_VERSION
+ARG TARGETARCH
+
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)